nebula/ttypes.go

```diff
@@ -233,13 +280,23 @@ const (
   ErrorCode_E_USER_NOT_FOUND ErrorCode = -18
   ErrorCode_E_STATS_NOT_FOUND ErrorCode = -19
   ErrorCode_E_SERVICE_NOT_FOUND ErrorCode = -20
+  ErrorCode_E_DRAINER_NOT_FOUND ErrorCode = -21
+  ErrorCode_E_DRAINER_CLIENT_NOT_FOUND ErrorCode = -22
+  ErrorCode_E_PART_STOPPED ErrorCode = -23
   ErrorCode_E_BACKUP_FAILED ErrorCode = -24
   ErrorCode_E_BACKUP_EMPTY_TABLE ErrorCode = -25
   ErrorCode_E_BACKUP_TABLE_FAILED ErrorCode = -26
   ErrorCode_E_PARTIAL_RESULT ErrorCode = -27
   ErrorCode_E_REBUILD_INDEX_FAILED ErrorCode = -28
   ErrorCode_E_INVALID_PASSWORD ErrorCode = -29
   ErrorCode_E_FAILED_GET_ABS_PATH ErrorCode = -30
+  ErrorCode_E_LISTENER_PROGRESS_FAILED ErrorCode = -31
+  ErrorCode_E_SYNC_LISTENER_NOT_FOUND ErrorCode = -32
+  ErrorCode_E_DRAINER_PROGRESS_FAILED ErrorCode = -33
+  ErrorCode_E_PART_DISABLED ErrorCode = -34
+  ErrorCode_E_PART_ALREADY_STARTED ErrorCode = -35
+  ErrorCode_E_PART_ALREADY_STOPPED ErrorCode = -36
+  ErrorCode_E_QUERY_TIMEDOUT ErrorCode = -37
```

Comment:
Are all these error codes currently missing but should have existed?

Reply:
internel error code
e.g. graphd won't send `ErrorCode_E_PART_DISABLED` code to client